app.test.js

```diff
@@ -56,9 +56,9 @@ describe("GET /api/articles/:article_id", () => {
           author: "butter_bridge",
           body: "I find this existence challenging",
           created_at: "2020-07-09T20:11:00.000Z",
-          votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: expect.any(String),
```

Comment:
Because you have used [toMatchObject](https://jestjs.io/docs/expect#tomatchobjectobject)  here there is no need to add this property to this test. You have made new tests that cover this new property so all good. In fact it is bad practise to alter passing tests 👀 😄